storage-stack-automation/README.md

```diff
@@ -0,0 +1,36 @@
+# Automation using Lambda to Deploy FSS- Storage Stack 
+
+Automated process to deploy FSS Storage stack on each new S3 bucket using Lambda. The storage stack will be linked to the scanner stack previously defined.
+
+![](architecture.png)
+# Deploy via CloudFormation
+
+   * If not already present, [deploy a Scanner Stack](https://cloudone.trendmicro.com/docs/file-storage-security/stack-add/#AddScanner) in your Cloud One Account
```

Comment:
* If not already present, [deploy a Scanner Stack](https://cloudone.trendmicro.com/docs/file-storage-security/stack-add/#AddScanner) in the Cloud One - File Storage Security account.

Reply:
Fixed